server/src/main/java/keywhiz/service/permissions/KeywhizPrincipal.java

```diff
@@ -0,0 +1,4 @@
+package keywhiz.service.permissions;
+
+public interface KeywhizPrincipal {
+}
```

Comment:
Now I'm wondering if we should put this into the auth package, or leave it here and call it "Actor". Let's leave it as-is and we can figure out the naming later.